torchtune/utils/metric_logging.py

```diff
@@ -166,9 +176,29 @@ def __init__(
             group=group,
             reinit=True,
             resume="allow",
-            config=kwargs,
+            **kwargs,
         )
 
+    def log_config(self, config: OmegaConf) -> None:
+        "Logs the config to W&B. Also updates config on overview tab."
+
+        resolved = OmegaConf.to_container(config, resolve=True)
+        self._wandb.config.update(resolved)
+        
+        output_config_fname = Path(os.path.join(
+            config.checkpointer.checkpoint_dir, 
```

Comment:
Can we assume that this field will always exist in the config? I think it's fair to say this is required for any fine-tuning recipe, but I wonder if you'd ever want to log the config to wandb in non-finetuning recipes? Or maybe we should just pass the checkpoint dir as a parameter? cc @kartikayk